src/components/header.svelte

```diff
@@ -4,19 +4,27 @@
     import Burger from "./burger.svelte";
     import { page } from "$app/stores";
     import { activeSectionId, lightModeNavbar } from "$lib/store";
+    import { onMount } from "svelte";
+
     export let items = [];
     export { classes as class };
 
     let classes: string = "";
-
+    let scroll: number;
     let sideMenuOpen: boolean = false;
 
+    onMount(() => {
+        window.addEventListener("scroll", () => {
```

Comment:
you need to unsubscribe from the eventListener onDestroy of the component. You can do that here in the return callback of `onMount` or in `onDestroy`